package.json

```diff
@@ -47,7 +47,7 @@
     "method-override": "~2.3.5",
     "gulp-wiredep": "~0.0.0",
     "mocha": "~2.4.5",
-    "mongoose": "~4.4.3",
+    "mongoose": "~4.4.3 <4.4.7",
```

Comment:
This shouldn't be in this PR. This diff was already merged into master. See https://github.com/meanjs/mean/pull/1264. It fixed the issue with our builds failing.